src/ErrorNotification.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useState } from 'react';
+import classnames from 'classnames';
+
+type Props = {
+  error: string;
+};
+
+export const ErrorNotification:React.FC<Props> = ({ error }) => {
+  const [hidden, setHidden] = useState(false);
+
+  const autoHideError = useCallback(
+    () => {
+      setTimeout(() => {
+        setHidden(true);
+      }, 3000);
+    }, [],
+  );
+
+  autoHideError();
```

Comment:
The autoHideError function is called directly in the component body, which may cause multiple timeouts to be set during re-renders. Consider using useEffect to call this function only on initial render or when the error prop changes.